Formula/rna-star.rb

```diff
@@ -0,0 +1,35 @@
+class RnaStar < Formula
+  desc "RNA-seq aligner"
+  homepage "https://github.com/alexdobin/STAR"
+  # cite "https://doi.org/10.1093/bioinformatics/bts635"
+
+  url "https://github.com/alexdobin/STAR/archive/2.7.3a.tar.gz"
+  version "2.7.3a"
+  sha256 "de204175351dc5f5ecc40cf458f224617654bdb8e00df55f0bb03a5727bf26f9"
+  head "https://github.com/alexdobin/STAR.git"
+
+  # Fix error: 'omp.h' file not found
+  depends_on "gcc"
```

Comment:
Is this line needed? I thought that `libomp` provided `omp.h` on macOS? And `gcc` is the default on Linux.